shell/platform/darwin/ios/BUILD.gn

```diff
@@ -153,6 +146,146 @@ shared_library("create_flutter_framework_dylib") {
   ]
 }
 
+ocmock_path = "../../../../../third_party/ocmock/Source"
+
+# TODO: Clone the OCMock repository so we can add a BUILD.gn to it.
+static_library("ocmock") {
```

Comment:
This definition goes in `//build/secondary/third_party/ocmock/BUILD.gn`. Thats where we maintain target build rules for dependencies that don't maintain their own GN rules (or said rules are incompatible with our buildroot.)

Reply:
Also, while you at it, make this a `source_set` as that skips the archiving step.

Reply:
This needs to be a static_library because it is linked by xcode's build system when building the testing bundle.

Reply:
Thanks for clarifying.

Reply:
wrt moving the build file.  I added those details to the TODO about cloning the repository if we find we can't use chromium's clone.  I'll follow up on this, like the other issue I'd like to do anything that involves an extra repo for another PR since i'm juggling this and LUCI already.